### PR TITLE
Check to validate stack tag values

### PR DESCRIPTION
**Pull request by zaro0508**
Add a hook to check that the assigned values
to stack tags are valid.

```
➜ pre-commit run --all-files
[INFO] Initializing environment for https://github.com/zaro0508/sceptrelint.
[INFO] Installing environment for https://github.com/zaro0508/sceptrelint.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
check stack tag values...................................................Failed
- hook id: check-stack-tag-values
- exit code: 1

- config/prod/htan-dcc-hms.yaml: "GHTAN-DFCI / 120100" is not a valid CostCenter
- config/prod/AMP-Emory-Bucket.yaml: "QTREAT AD - Emory / 121100" is not a valid CostCenter
```

### CODE REVIEW

**Review by brucehoff — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -24,14 +24,28 @@ CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/c
...
 ```
 
+* `check-stack-tag-values` Checks that a specific stack tag is assigned a valid value.
+
+The below checks that the `CostCenter` tag is defined in sceptre config's `stack_tags`
+key and that the value assigned to it is valid.  The valid tag values are passed
+in with a `file`.
+```yaml
+-   id: check-stack-tags
+    args: [--tag=CostCenter, --file=/path/to/file.json]
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply README.md:43]: yes, i can document that.  I do prefer the shorter `--file` flag though.

File: tests/check_stack_tag_values_test.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+def test_stack_tags_key_exist_value_valid_single_file():
+    resource_path = get_resource_path(TEST_RESOURCES_DIR)
+    files = [f'{resource_path}/stack_tags_match_tag_values1.yaml']
+    assert not lint(files, 'color', [f'{resource_path}/tag_values1.json'])
+
+
+def test_stack_tags_key_exist_value_valid_multiple_files():
+    resource_path = get_resource_path(TEST_RESOURCES_DIR)
+    files = [f'{resource_path}/stack_tags_match_tag_values2.yaml']
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply tests/check_stack_tag_values_test.py:17]: yes, the test is to verify that `green` is valid.  Are you just making a comment or pointing out something that's not correct?

[the remark this answers is not in the document]
Comment (brucehoff) [reply tests/check_stack_tag_values_test.py:17]: I was just making a comment as I was trying to follow what the test was doing.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zaro0508) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -24,14 +24,28 @@ CloudFormation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/c
...
+```
+
+__Note__: The `--file` flag can take a local or a url reference (i.e. https://raw.githubusercontent.com/acme/repo/master/tags/cost_center_codes.json). The `--file` flage can be use multiple times to
```
[the remark this answers is not in the document]
Comment (zaro0508) [reply README.md]: fixed

**Review by brucehoff — APPROVED** [review]

**Review by brucehoff — APPROVED** [review]

**Pull request merged**